### PR TITLE
Set top edge to 0 for fixed position banner ads.

### PR DESCRIPTION
**Pull request by Ap0c**
## What does this change?

Sets top edge to 0 on banner ads in immersive galleries.
## What is the value of this and can you measure success?

It fixes a compatibility problem in IE11/Edge, where banner ads were appearing further down the page in immersive galleries and covering other content.
## Screenshots

**Edge Before**

edge-before

**Edge After**

edge-after
## Request for comment

@regiskuckaertz

### CODE REVIEW

File: static/src/stylesheets/module/_adslot.scss [hunk 1]
```diff
@@ -165,6 +165,12 @@
     }
 }
 
+// Compatibility fix for top banner ads in IE11/Edge. Without this top banner
+// appears further down the page, and covers content, in immersive galleries.
+.immersive-header-container .top-banner-ad-container {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/_adslot.scss]: Would it do any harm to add the `top` to the top banner everywhere, not only in immersive galleries? Let's try to keep the layout consistent.

Comment (Ap0c) [inline static/src/stylesheets/module/_adslot.scss]: Ok sure, tweaked that to cover all .top-banner-ad-container elements.

**Comment by regiskuckaertz** [comment]
👍

**Comment by regiskuckaertz** [comment]
You will need to merge master to remove expired switches.

**Comment by paperboyo** [comment]
Thank you, Gentlemen!

**Pull request merged**